README.md

```diff
@@ -63,6 +63,45 @@ public void testAssertOnUnhandledException() throws InterruptedException {
 }
 ```
  
+## JUnit 5
+ You can use it for JUnit 5 test like this:
```

Comment:
In case of using Junit 5, You can use UncaughtExceptionExtension extension like this: